tasks/config.yml

```diff
@@ -385,3 +385,85 @@
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^defaultMeetingDuration='
+    line: 'defaultMeetingDuration={{ bbb_default_meeting_duration }}'
+  notify: restart bigbluebutton
+
+- name: Set the maximum number of pages allowed for an uploaded presentation
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^maxNumPages='
+    line: 'maxNumPages={{ bbb_max_num_pages }}'
+  notify: restart bigbluebutton
+
+- name: Set the max allowed number of minutes the conversion should take
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^maxConversionTime='
+    line: 'maxConversionTime={{ bbb_max_conversion_time }}'
+  notify: restart bigbluebutton
+
+- name: Set the number of threads in the pool to do the presentation conversion
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^numConversionThreads='
+    line: 'numConversionThreads={{ bbb_num_conversion_threads }}'
+  notify: restart bigbluebutton
+
+- name: Set the number of threads to process file uploads
+  lineinfile:
+    path: /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
+    regexp: '^numFileProcessorThreads='
+    line: 'numFileProcessorThreads={{ bbb_num_file_processor_threads }}'
+  notify: restart bigbluebutton
+
+- name: Set the retention period for presentations, red5, kurento, and freeswitch caches
+  lineinfile:
+    path: /etc/cron.daily/bigbluebutton
+    regexp: '^history'
+    line: 'history={{ bbb_cron_history }}'
+
+- name: Set the retention period of recordings for meetings with no recording markers
+  lineinfile:
+    path: /etc/cron.daily/bigbluebutton
+    regexp: '^unrecorded_days'
+    line: 'unrecorded_days={{ bbb_cron_unrecorded_days }}'
+
+- name: Set the retention period of recordings’ raw data
+  lineinfile:
+    path: /etc/cron.daily/bigbluebutton
+    regexp: '^published_days'
+    line: 'published_days={{ bbb_cron_published_days }}'
+
+- name: Set the retention period of old log files
+  lineinfile:
+    path: /etc/cron.daily/bigbluebutton
+    regexp: '^log_history'
+    line: 'log_history={{ bbb_cron_log_history }}'
+
+- name: Disable muted sound
+  replace:
+    path: /opt/freeswitch/etc/freeswitch/autoload_configs/conference.conf.xml
+    regexp: '<param name="muted-sound" value="conference/conf-muted.wav"/>'
+    replace: '<!-- <param name="muted-sound" value="conference/conf-muted.wav"/> -->'
+  when: bbb_freeswitch_muted_sound == false
+  notify: restart bigbluebutton
+
+- name: Disable unmuted sound
+  replace:
+    path: /opt/freeswitch/etc/freeswitch/autoload_configs/conference.conf.xml
+    regexp: '<param name="unmuted-sound" value="conference/conf-unmuted.wav"/>'
+    replace: '<!-- <param name="unmuted-sound" value="conference/conf-unmuted.wav"/> -->'
+  when: bbb_freeswitch_unmuted_sound == false
+  notify: restart bigbluebutton
```

Comment:
This RegEx is one sided.
If this is deployed once with `bbb_freeswitch_unmuted_sound: false` a re-deploy with `bbb_freeswitch_unmuted_sound: true` will have no effect.
An extra step might be needed for the `true` case.

My propose to add a section:

```yaml
- name: Enable unmuted sound
  replace:
    path: /opt/freeswitch/etc/freeswitch/autoload_configs/conference.conf.xml
    regexp: '<!-- <param name="unmuted-sound" value="conference/conf-unmuted.wav"/> -->'
    replace: '<param name="unmuted-sound" value="conference/conf-unmuted.wav"/>'
  when: bbb_freeswitch_muted_sound == true
  notify: restart bigbluebutton
```